app/controllers/api/v2/providers_controller.rb

```diff
@@ -79,18 +79,24 @@ def suggest_any
         return render(status: :bad_request) if params[:query].nil? || params[:query].length < 2
         return render(status: :bad_request) unless begins_with_alphanumeric(params[:query])
 
-        found_providers = @recruitment_cycle.providers
-          .search_by_code_or_name(params[:query])
-          .limit(5)
+        scope = @recruitment_cycle.providers
+                                  .search_by_code_or_name(params[:query])
+                                  .limit(5)
+
+        scope = scope.are_accredited_bodies if accredited_body_filter?
```

Comment:
```suggestion
        scope = scope.accredited_body if accredited_body_filter?
```

https://github.com/DFE-Digital/teacher-training-api/blob/fdb35d6428d7ad39ad570fe39cba2086ebde8318/app/models/provider.rb#L23-L26

Reply:
nice, i wasn't aware it made a scope too